### PR TITLE
[#211] Add custom error pages

### PR DESCRIPTION
**Pull request by victorphoenix3**
Error pages created:

- [x] 400
- [x] 403
- [x] 403 CSRF Token invalid
- [x] 404
- [x] 500

Fixes: #211

Signed-off-by: Jayati Shrivastava <gaurijove@gmail.com>

### CODE REVIEW

**Comment by tristanlatr** [comment]
Hi @victorphoenix3, thanks a lot for this, it shall be useful I think.

Just a question: Does it replaces the default error pages even when `DEBUG=True` ?
I think it would be good to keep the verbose error pages for development. 

Tell me what you think,

Thanks

**Comment by victorphoenix3** [comment]
@tristanlatr No, with `DEBUG=True`, we get the default django error pages

File: saskatoon/sitebase/templates/app/errors/400.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% extends 'app/errors/error.html' %}
+{% load i18n %}
+{% load static %}
+
+{% block content %}
+    <div class="container">
+        <div class="row" style="margin-top:1rem;">
+            <div class="col text-center">
+                <h1> 400 Bad Request</h1>
```
Comment (tristanlatr) [inline saskatoon/sitebase/templates/app/errors/400.html:9]:
```suggestion
                <h1> {% translate "400 Bad Request" %} </h1>
```

**Review by tristanlatr — CHANGES_REQUESTED** [review]
Hi @victorphoenix3, 

I was going to merge this PR when I realized that we could use translations there too, that would be good. So there are accompanying suggestions to add the translate fucntion.

Thanks a lot,

**Comment by tristanlatr** [comment]
Hi @victorphoenix3, could you update this branch with develop ?

Thanks,

**Comment by victorphoenix3** [comment]
@tristanlatr done

**Review by 2ynn — APPROVED** [review]
great thanks!

**Review by tristanlatr — APPROVED** [review]

**Pull request merged**